addons/mrp/mrp.py

```diff
@@ -1016,6 +1016,17 @@ def action_produce(self, cr, uid, production_id, production_qty, production_mode
         self.signal_workflow(cr, uid, [production_id], 'button_produce_done')
         return True
 
+    def button_unreserve(self, cr, uid,  production_ids, context=None):
+        #import pudb
+        #pudb.set_trace()
+        productions = self.read(cr, uid, production_ids, ['move_lines', 'move_created_ids'], context=context)
+        self.pool.get('stock.move').action_cancel(cr, uid, productions[0]['move_lines'], context=context)
```

Comment:
hm, indeed, but you will create a lot of problems when these moves where MTO.   I would prefer to rethink it for master than to do bad things here.